### ISSUE TITLE
Call for votes: New swag budget proposal

### ISSUE THREAD

**Issue by benjagm** [json-schema-org/TSC#8]
## Context

This issue is to ask the TSC for vote to approve/reject the new Swag budget detailed in this issue comment: https://github.com/json-schema-org/community/issues/678#issuecomment-2114568119. 

The reason for a new budget is because after looking into the possible high delivery and customs costs we thought that using a global print-on-demand provider will help to keep those costs reasonably low and also remove the swag storage costs. As a result my final proposal is to move forward with this new approach focused on using Shopify and Printful as print-on-demand provider. 

In addition will need to pay an Illustrator/Designer to produce all the designs as part of the shop setup. This cost is also included in the proposal.

## In summary

A 8K budget was already approved here #4 , and this new call for votes is to make sure we have the support of the leading team for a 50% reduction the estimated costs but also pay an Illustrator/Designer to produce the swag designs.

(Discussed in Open Community Working Meeting 2024-04-08 https://github.com/json-schema-org/community/issues/688)

See voting process [here](https://github.com/json-schema-org/community/blob/main/GOVERNANCE.md#decision-making-via-vote).

## Related

Replaces #4

**Comment by benjagm** [comment]
Please signal your vote to proceed with this new proposal.

👍 In Favour
👎 Against

**Comment by benjagm** [comment]
Thanks everyone. This is the result:

- Vote meets quorum threshold.
- Vote meets time requirements.
- Votes are 5 TSC members in favour.
